src/router.js

```diff
@@ -2,7 +2,16 @@ import { createRouter, createWebHistory } from "vue-router";
 
 import Login from "./views/Login.vue";
 import ResumeView from "./views/Resume/ResumeList.vue";
-import InformationView from "./views/Information/InformationList.vue";
+import InformationList from "./views/Information/InformationList.vue";
+import InfoExperience from "./components/InfoExperience.vue"
+import InfoSkills from "./components/InfoSkills.vue"
+import InfoAwards from "./components/InfoAwards.vue"
+import InfoLinks from "./components/InfoLinks.vue"
+import InfoEducation from "./components/InfoEducation.vue"
+import InfoProjects from "./components/InfoProjects.vue"
+import InfoUsers from "./components/InfoUsers.vue"
```

Comment:
These should link directly to their respective ./components/(category)/(category)View.vue (ie. /Links/LinksView.vue)